packages/kit/src/runtime/client/renderer.js

```diff
@@ -369,8 +369,18 @@ export class Renderer {
 			const { scroll, keepfocus } = opts;
 
 			if (!keepfocus) {
+				const root = document.documentElement;
```

Comment:
can we add some comment here about what this is doing and why

Reply:
pushed some comments